packages/yoroi-extension/package.json

```diff
@@ -177,7 +177,7 @@
     "@emurgo/cardano-serialization-lib-browser": "12.0.0-alpha.15",
     "@emurgo/cip14-js": "2.0.0",
     "@emurgo/cip4-js": "1.0.5",
-    "@emurgo/cross-csl-browser": "4.0.0-alpha.1",
+    "@emurgo/cross-csl-browser": "4.1.0",
```

Comment:
Can't merge it just like that. We are now restricted to always using the matching cross-csl version with the `yoroi-eutxo-txs`. First a version of eutxo needs to be published that used cross-csl `4.1.0` and then we can update both eutxo and cross-csl together